packages/plugin-commands-init/src/init.ts

```diff
@@ -54,6 +55,11 @@ export async function handler (
     author: '',
     license: 'ISC',
   }
+  const memReadLocalConfig = mem(readLocalConfig)
+  const localConfig = await memReadLocalConfig(process.cwd())
+  if (!localConfig.initPackageManager) {
+    delete manifest.packageManager
+  }
```

Comment:
This is not needed. The init settings are set globally.

Reply:
Just need to support `pnpm config set init-package-manager=true/false`?

Reply:
yes